assets/sass/pages/_login.scss

```diff
@@ -0,0 +1,12 @@
+.login-section {
```

Comment:
Like Jaakko said in the profile page PR #688, these page-specific rules could be nested under a page-specific selector. Currently, the body element has the attribute `data-view-tag` that can be set in the HTML template with block view_tag. The CSS could use rule like `body[data-view-tag=login]`.

Reply:
In this particular case, I do not see the need for adding a parent selector since the `login-section` class is specific to the login page. This class is not used in any other part of the application.